src/elexmodel/models/BaseElectionModel.py

```diff
@@ -33,7 +40,16 @@ def fit_model(self, model, df_X, df_y, tau, weights, normalize_weights):
         X = df_X.loc[:, columns_to_keep].values
         y = df_y.values
         weights = weights.values
-        model.fit(X, y, tau_value=tau, weights=weights, normalize_weights=normalize_weights)
+
+        # normalizing weights speed up solution by a lot. However, if the relative size
+        # of the smallest weight is too close to zero, it can lead to numerical instability
+        # where the solver either throws a warning for inaccurate solution or breaks entirely
+        # in that case, we catch the error and warning and re-run with normalize_weights false
+        try:
```

Comment:
Should your example command fail/error from the develop branch? Seems to run OK for me